Misc/NEWS.d/next/Tools-Demos/2022-08-11-09-58-15.gh-issue-64490.PjwhM4.rst

```diff
@@ -0,0 +1,3 @@
+Fix an out-of-bound bug of :c:func:`_PyArg_UnpackKeywordsWithVararg`.
```

Comment:
```suggestion
Fix an out-of-bound bug of ``_PyArg_UnpackKeywordsWithVararg``.
```
It is undocumented so linking will not work.

Reply:
> It is undocumented so linking will not work.

True, but literals are rendered differently from functions. I would instead do this:

```suggestion
Fix an out-of-bound bug of :c:func:`!_PyArg_UnpackKeywordsWithVararg`.
```

Also, I think the NEWS entry needs a little bit more tweaking before landing this.